miden-lib/build.rs

```diff
@@ -97,12 +152,17 @@ fn main() -> io::Result<()> {
         _ => (),
     }
 
+    // compile the stdlib
     let namespace =
         LibraryNamespace::try_from("miden".to_string()).expect("invalid base namespace");
     let version = Version::try_from(env!("CARGO_PKG_VERSION")).expect("invalid cargo version");
-    let stdlib = MaslLibrary::read_from_dir(dst.join(ASM_DIR_PATH), namespace, false, version)?;
+    let stdlib =
+        MaslLibrary::read_from_dir(dst.join(ASM_MIDEN_DIR_PATH), namespace, true, version)?;
 
     stdlib.write_to_dir(Path::new(&build_dir).join(ASL_DIR_PATH))?;
 
+    // compile the note scripts separately because they are not part of the stdlib
+    compile_note_scripts(dst)?;
```

Comment:
We should probably create an equivalent function for the code above - e.g., something like `compile_miden_lib()`.

Reply:
jup